Dockerfile

```
@@ -112,22 +108,12 @@ RUN apt-get update && \
     && \
     apt-get clean
 
-RUN git clone --depth=1 --single-branch --branch=frankenphp-8.2 https://github.com/dunglas/php-src.git && \
-    cd php-src && \
-    #export CFLAGS="-DNO_SIGPROF" && \
-    # --enable-embed is only necessary to generate libphp.so, we don't use this SAPI directly
-    ./buildconf && \
-    ./configure \
-        --enable-embed=static \
-        --enable-zts \
-        --disable-zend-signals \
-        --with-config-file-path="$PHP_INI_DIR" \
-        --with-config-file-scan-dir="$PHP_INI_DIR/conf.d" && \
-    make -j$(nproc) && \
-    make install && \
-    rm -Rf php-src/ && \
-    ldconfig && \
-    php --version
+COPY --from=php-base /usr/local/include/php/ /usr/local/include/php
+COPY --from=php-base /usr/local/lib/libphp.* /usr/local/lib
+COPY --from=php-base /usr/local/lib/php/ /usr/local/lib/php
+COPY --from=php-base /usr/local/php/ /usr/local/php
+COPY --from=php-base /usr/local/bin/ /usr/local/bin
+COPY --from=php-base /usr/src /usr/src
```

We need to double-check that these directories are empty in the go image then.

A future-proof solution would be to copy these directories in a different location and "merge" both.

Something like:

```suggestion
COPY --from=php-base /usr/local/bin/ /usr/local/php-bin
COPY --from=php-base /usr/src /usr/php-src

RUN cp -R /usr/local/php-bin/* /usr/local/bin/ && \
  cp -R /usr/php-src/* /usr/src && \
  rm -Rf /usr/local/php-bin /usr/php-src
```

(not tested)

---

I believe it should already do the merging. I could be wrong, but it should "just work"